lib/bridge.js

```diff
@@ -18,6 +18,7 @@ const Promise = require("bluebird");
 const Datastore = require("nedb");
 const util = require("util");
 const MembershipCache = require("./components/membership-cache");
+const log = require("./components/logging").Get("Bridge");
```

Comment:
general principle: the log name should match the file name including casing (`bridge`, `cli`, etc)